llvm/test/CodeGen/AArch64/store.ll

```diff
@@ -2,9 +2,6 @@
 ; RUN: llc -mtriple=aarch64 %s -o - | FileCheck %s --check-prefixes=CHECK,CHECK-SD
 ; RUN: llc -mtriple=aarch64 -global-isel -global-isel-abort=2 %s -o - 2>&1 | FileCheck %s --check-prefixes=CHECK,CHECK-GI
```

Comment:
If there is no longer any fallbacks, this could remove the -global-isel-abort=2 (and 2>&1). That might be easier to do if you land the initial tests separately